013/src/backEnd/view/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="../../../build/backEnd/static/main.css" />
```

Comment:
```suggestion
    <link rel="stylesheet" href="/main.css" />
```
이렇게 불필요한 경로를 클라이언트에 노출시키지 않도록 작업해주세요